pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
   <div class="hero-body">
```

Comment:
With all due respect, but I think this is quite a setback for the homepage. Horizontally centring of longer stretches of text makes them very hard to read, and it just does not feel as a cohesive whole - the first impression is of a somewhat amateurish website, IMO. I'd prefer to try to fit the new content into the existing design Lisa made earlier, and possibly to ask her (or someone else) to update the design based on the new content, rather than trying to cobble something together quickly that looks the part.

(Also sorry that I'm being a bit rough here; if we didn't have the time pressure I could provide a more constructive path forward, but in it's current state, I don't think it's actually an improvement over what we have.)

Reply:
Also, aren't we writing "Pod" with a capital P? :)

Reply:
Definitely not meant to be the long-term design. Already have a request in for design help, but wanted to get the content updated first, and didn't have the corresponding graphics available for it. Will see if something can be done today.

Reply:
So how about replacing the text that currently accompany the illustrations by context you've added, and simply removing the buttons? Even if the illustrations do not match the text, it'd still look a whole lot more professional?

Reply:
I'm talking through this with @tony-morelli now - but i agree and i think we can settle it today. Main question is whether or not to push this in the meantime and then update in an iteration tonight or tomorrow.